### ISSUE TITLE
Delete /tmp/whateverable/rakudo-moar/* builds older than max possible timeout

### ISSUE THREAD

**Issue by MasterDuke17** [perl6/whateverable#144]
See https://irclog.perlgeek.de/perl6-dev/2017-05-19#i_14607655 for example of the problem. We could probably just delete any builds where the creation date of the folder is older than the max TOTAL-TIME allowed for any of the bots to run. Do it here (https://github.com/perl6/whateverable/blob/master/Whateverable.pm6#L168-L176) maybe?

**Comment by AlexDaniel** [comment]
Yes, the line numbers are correct. That's exactly where I would expect it to happen.

**Comment by zoffixznet** [comment]
(permalink to the last link in OP: https://github.com/perl6/whateverable/blob/09c46d5915edfca48944fc8854dc85697cdf0d3d/Whateverable.pm6#L168-L176 )

**Comment by MasterDuke17** [comment]
We'll need to use the `--touch` tar option when extracting builds, otherwise the file creation date will be when the build was originally created, which will almost always be much older than the max possible timeout.

**Comment by AlexDaniel** [comment]
Ah. This is no longer a problem after https://github.com/perl6/whateverable/commit/845db2e47fd6468a069bc9f2b54997e8ae412f48. Every bot has its own private ``/tmp`` which is wiped when the service is restarted.

**Comment by AlexDaniel** [comment]
OK it may be a problem for the bot itself if it left something behind and never exited, but so far this has never happened. Non-issue really, but not sure how to test it.

**Comment by AlexDaniel** [comment]
I'd call this unfixed. During the squashathon some people reported stuff being left behind, so it is still a problem (even if it doesn't affect the bots on the server). See also #122.
